### ISSUE TITLE
addInfiniteScroll is not called when horizontal collection view scrolled till end.

### ISSUE THREAD

**Issue by monica18p** [pronebird/UIScrollView-InfiniteScroll#48]
Hi Pronebird,
I have a horizontal collectionview i.e. it could be scrolled only horizontally.
```
self.view.addSubview(self.collectionView!)
collectionView?.addInfiniteScroll(handler: { [weak self] (scrollView) in
            let collectionView = scrollView
            ....
.....
            
        })
```
This closure is not called when collection view is scrolled till end.

**Comment by pronebird** [comment]
Hi @monica18p,

So far it only supports vertical flow, but it would great to see horizontal behavior implemented.

**Comment by monica18p** [comment]
Alrighty!

**Comment by pronebird** [comment]
Fixed in #71
